src/google_map.js

```diff
@@ -266,8 +266,7 @@ export default class GoogleMap extends Component {
       );
     }
 
-    addPassiveEventListener(window, 'mouseup', this._onChildMouseUp, false);
-    const bootstrapURLKeys = {
+    addPassiveEventListener(window, 'mouseup', this._onChildMouseUp, false);    const bootstrapURLKeys = {
```

Comment:
I think you messed up here by moving `bootstrapURLKeys` const to the previous line, if you change this I am happy to merge!

Reply:
Not sure how that happened. Oh well, it's done :)

Reply:
No worries, thank you so much!